R/qenv-get_code.R

```diff
@@ -3,6 +3,7 @@
 #' @name get_code
 #' @param object (`qenv`)
 #' @param deparse (`logical(1)`) if the returned code should be converted to character.
+#' @param name (`character(1)`) if provided, returns the code only for the object specified in `name`.
```

Comment:
Needs to be plural. We need to be able to extract code of multiple datasets as one code output.
```
testthat("Possible to output code for multiple datasets", {
  q <- new_qenv()
  q <- eval_code(q, "a <- 1")
  q <- eval_code(q, "b <- 2")
  q <- eval_code(q, "c <- 3")

  testthat::expect_identical(
    get_code(q, deparse = FALSE, name = c("a", "b")),
    c("a <- 1", "b <- 2")
  )
})
```

Reply:
good idea. this is now possible